server/src/main/resources/log4j2.xml

```diff
@@ -26,8 +26,8 @@
 
         <RollingFile
                 name="logfile"
-                fileName="/var/log/tomcat8/stocks.log"
-                filePattern="/var/log/tomcat8/stocks-%i.log">
+                fileName="logs/stocks.log"
+                filePattern="logs/stocks-%i.log">
```

Comment:
Please also change the section on logging in `manual/server/maintainance.md` [sic].

Reply:
Oh and please adapt the bash scripts you find by `grep -r /var/log/tomcat8 --exclude-dir target -I`.
They collect logs from CI and make them available in the GUI as build artifacts.

Reply:
Will do, but probably only after I fixed the builds tomorrow.

Reply:
Will stay like this as mainly Arch with tomcat8 cares about these files.